client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -57,6 +57,20 @@ const Task = ( {
 	const dismissalPreferenceKey = `dismissible-card-home-task-${ taskId }-${ siteId }`;
 	const successNoticeId = `task_remind_later_success-${ taskId }`;
 
+	const startTask = () => {
+		if ( actionOnClick instanceof Function ) {
+			actionOnClick();
+		}
+		dispatch(
+			composeAnalytics(
+				recordTracksEvent( 'calypso_customer_home_task_start', {
+					task: taskId,
+				} ),
+				bumpStat( 'calypso_customer_home', 'task_start' )
```

Comment:
If tasks happen to dispatch a redux action onClick it'd be nice to bundle this together, but we can leave that for the future if this happens.